File: examples/forex/price/node/getAnFxPrice.js
const { ForexService } = require("m3o/forex");

// Get the latest price for a given forex ticker
async function getAnFxPrice() {
  let forexService = new ForexService(process.env.MICRO_API_TOKEN);
  let rsp = await forexService.price({
    symbol: "GBPUSD",
  });
  console.log(rsp);
}

getAnFxPrice();

File: examples/go/forex/price/getAnFxPrice.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/forex"
)

// Get the latest price for a given forex ticker
func GetAnFxPrice() {
	forexService := forex.NewForexService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := forexService.Price(&forex.PriceRequest{
		Symbol: "GBPUSD",
	})
	fmt.Println(rsp, err)
}
